docs/ansible/roles/prometheus_server/getting-started.rst

```diff
@@ -15,16 +15,16 @@ Default configuration
 ---------------------
 
 The configuration is split into 6 basic parameters,
-this is because of limitation of YAML and easier representation.
+this is because of YAML limitations and easier representation.
 
-- prometheus_server__global_configuration
-- prometheus_server__rule_files_configuration
-- prometheus_server__alerting_configuration
-- prometheus_server__*_scrape_configs_configuration
-- prometheus_server__remote_write_configuration
-- prometheus_server__remote_read_configuration
+- :envvar:`prometheus_server__global_configuration`
+- :envvar:`prometheus_server__rule_files_configuration`
+- :envvar:`prometheus_server__alerting_configuration`
+- :envvar:`prometheus_server__*_scrape_configs_configuration`
```

Comment:
@ypid This envvar doesn't exist. The envar is prometheus_server__scrape_configs_configuration and docs script fails. The combined envar is only for support default node and alertmanager scrape configuration. Maybe is better set prometheus_server__scrape_configs_configuration here.

What do you think¿

Reply:
You are right. Sorry for that. Making `prometheus_server__*_scrape_configs_configuration` an `envvar` was pretty stupid of me.

I would propose this:

```RST
:envvar:`prometheus_server__combined_scrape_configs_configuration`
```

That describes the `*` best I would say. Also, that is the common reference other roles use in `defaults-detailed.rst`.

Reply:
Fixed! Thanks for the PR :)

Reply:
You are welcome :)